docs/workflow.md

```diff
@@ -0,0 +1,137 @@
+
+# Workflows
+
+Ths document documents two different yet common workflows in BKR using `kubeprod` and `kubecfg`:
+
+* Basic workflow
+* [Advanced workflow](#advanced-workflow)
+
+## Pre-requisites
+
+* An existing Kubernetes cluster.
+* `kubeprod` binary from https://github.com/bitnami/kube-prod-runtime/releases
+
+## Advanced workflow
+
+1. Ensure the default `kubectl` context
```

Comment:
I would remove this list, as it is a bit confusing. I would just add the details that you added below on each of the sections